.gitignore

```diff
@@ -273,4 +273,6 @@ experiments/outputs/*
 *-ubyte
 *-ubyte.gz
 *.tar.gz
-*cifar-10-batches-py
\ No newline at end of file
+*cifar-10-batches-py
+
+!tests/test_dataset/obj_det_landmarks/train/data/images/*
```

Comment:
this ignore file would be better to be placed on your own local folder. fill it with "*" in gitignore would ignore everything, including the gitignore itself, so it would not added up in the main gitignore file

Reply:
but this one is intended to not ignore the images under the dataset, so i think it serve its purpose. CMIIW